rules-reviewed/hibernate/hibernate.windup.xml

```diff
@@ -176,8 +174,8 @@
             <javaclass references="net.sf.hibernate.{classname}" />
          </when>
          <perform>
-            <classification title="Hibernate 2.x" severity="mandatory" />
-            <hint effort="1" severity="mandatory" title="Hibernate type reference net.sf.hibernate.{classname}">
+            <classification title="Hibernate 2.x reference" severity="optional" />
+            <hint effort="1" severity="optional" title="Hibernate type reference net.sf.hibernate.{classname}">
```

Comment:
this IMHO is not optional as Hibernate 2 is very old and not maintain in any current project